packages/block-library/src/latest-posts/edit.js

```diff
@@ -483,19 +483,27 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 								.split( ' ', excerptLength )
 								.join( ' ' ) }
 							{ createInterpolateElement(
-								/* translators: excerpt truncation character, default …  */
-								__( ' … <a>Read more</a>' ),
+								sprintf(
+									/* translators: %1$s is a static string "Read more", and %2$s is the post title only visible to screen readers. */
```

Comment:
```suggestion
									/* translators: 1: The static string "Read more", 2: The post title only visible to screen readers. */
```
[See here](https://github.com/WordPress/wordpress-develop/blob/6.3/src/wp-admin/includes/class-theme-upgrader.php#L83) for examples of how to document multiple `sprintf()` tokens.